scripts/build.sh

```diff
@@ -41,26 +41,52 @@ function main() {
     project_dir=$(git rev-parse --show-toplevel)
     cd "$project_dir" || exit 1
 
+    echo "using $(cmake -version) at $(which cmake)"
+
     mkdir -p build
     cd build || exit 1
-    working "Building project"
+
     local prefix_arg
     prefix_arg=$1
-    prefix_arg=${prefix_arg:+"-DCMAKE_INSTALL_PREFIX=$prefix_arg"}
-    cmake -DCMAKE_BUILD_TYPE=Debug ${prefix_arg} -DJAEGERTRACINGC_COVERAGE=${COVERAGE:OFF} ..
+
+    if [ "${USE_HUNTER:-}" == "off" ]; then
+        # We won't be using Hunter so we need to get opentracing's c library
+        # installed too.
```

Comment:
How would you prefer I handle this dependency to support non-Hunter builds?

Reply:
I'm not sure yet, thinking about that. It depends a lot on whether you think apps would expect to link to `opentracing-c` then load `jaeger-client-c` at runtime, or if you see them linking to a `jaeger-client-c` that implements the `opentracing-c` APIs exposed in `opentracing-c` headers.

(The build doesn't produce a shared library yet. I'll deal with that at some later stage, far from urgent.)

You've pushed opentracing-c to hunter, so I guess you see it working more like the former? The app links to opentracing-c, and it loads a tracing implementation at runtime? I haven't looked closely at this part yet, as I've just been trying to get the build to work.

No strong opinions here anyway. I anticipate I'll be treating it as a unit of opentracing-c + jaeger-client-c anyway.